FLEDGE.md

```diff
@@ -108,6 +109,8 @@ The browser will remain in an interest group for only a limited amount of time.
 
 #### 1.2 Interest Group Attributes
 
+The `priority` is used to select which interest groups are shown when the number of interest groups are limited by the `perBuyerGroupLimits` attribute of the auction config. If not specified, a `priority` of `0.0` is assigned.
```

Comment:
It almost goes without saying, but perhaps include that interest groups with higher `priority` will be selected over ones with lower?

Reply:
Done